keanu-python/tests/test_coal_mining.py

```diff
@@ -0,0 +1,18 @@
+import pandas as pd
+import keanu as kn
+import examples
+
+
+def test_coalmining():
+    model = examples.coal_mining_model()
+
+    FILE = "data/coal-mining-disaster-data.csv"
+    data = pd.read_csv(FILE, names=["year", "count"]).set_index("year")
+    model.disasters.observe(data.values)
```

Comment:
It would be better to deduplicate with `coal_mining_model` here. The model should do one of the following:
1. tell you the training data file name (i.e. change line 9)
2. give you the training data (i.e. change line 10)
3. provide a method `observeDefaultData` (i.e. change line 11)

Reply:
My vote would probably be for option 2